js/main.js

```diff
@@ -182,3 +191,19 @@ document.addEventListener('click', function (event) {
   }
 
 });
+
+// addEventListener for 'submit' for entry form
+createEntryForm.addEventListener('submit', function (event) {
+  event.preventDefault();
+
+  var entryUrl = document.forms[1].entryUrl.value;
+  var entryTitle = document.forms[1].entryTitle.value;
+  var notes = document.forms[1].notes.value;
+
+  entryData.entry.entryTitle = entryTitle;
```

Comment:
Instead of reusing a variable containing an object like this, you should create an object in the function block, so that the variable will cease to exist after the function is done. You should also push your object to the `entries` property in your data model 👍